src/main/java/com/fundation/logic/view/PlayerJFrame.java

```diff
@@ -57,19 +56,19 @@
      */
     public PlayerJFrame(String path) {
 
-        playButton = new JLabel();
-        stopButton = new JLabel();
-        pauseButton = new JLabel();
-        vol = 50;
-        headerPanel = new JPanel();
+        playButton = new JButton();
+        stopButton = new JButton();
+        pauseButton = new JButton();
         playingPanel = new JPanel();
         bottomPanel = new JPanel();
         progressBar = new JSlider(0, 100, 0);
-        volumen = new JSlider(0, 100, vol);
+        volumeLevel = 50;
+        volumeSlider = new JSlider(JSlider.VERTICAL,0, 100, volumeLevel);
+        //JSlider slider = new JSlider(JSlider.VERTICAL, 0, 20, 4);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
 
```

Comment:
remove this line.

Reply:
The line is used to separate the declaration part from the rest of the code